python/ray/rllib/es/es.py

```diff
@@ -249,25 +250,22 @@ def _train(self):
             g.shape == (self.policy.num_params,) and
             g.dtype == np.float32 and
             count == len(noise_indices))
-        update_ratio = self.optimizer.update(-g + config["l2coeff"] * theta)
+        # Compute the new weights theta.
+        theta, update_ratio = self.optimizer.update(
+            -g + config["l2_coeff"] * theta)
+        # Set the new weights in the local copy of the policy.
+        self.policy.set_weights(theta)
```

Comment:
Before the call to `set_weights` was happening in `self.optimizer.update`. I changed the optimizer to just return the new `theta`.